views/pages/invite.njk

```diff
@@ -4,12 +4,21 @@
     <h1>{{ __('invite.title') }}</h1>
 
     <div class="w-4/5 block border border-gray-500 border-solid p-5">
-        <p>{{ __('invite.give_code') }}</p>
+        <p class="font-semi-bold">{{ __('invite.give_code') }}</p>
+        <p>{{ __('invite.give_code_details') }}</p>
         <p class="font-bold text-6xl">{{ applicationCode }}</p>
-        <!-- TODO Dynamically generate the URL, rather than pulling in from translation -->
-        <p class="font-bold">{{ medicalProfessionalUrl }}</p>
-        <p>{{ __('invite.medical_professional_instruction') }}</p>
+        <p>{{ __('invite.expires', '[PLACEHOLDER DATE]') }}</p>
```

Comment:
Can we move line 10 and 11 below the button?
It's secondary information.

Reply:
done